src/Compilers/VisualBasic/Test/Emit/ErrorHandling.vb

```diff
@@ -1037,6 +1037,51 @@ End]]>)
 ]]>)
         End Sub
 
+        <Fact>
+        Public Sub SynthesizingDefaultConstructorsWithMsCorLibMissing_Nothing()
+            Dim comp = CompilationUtils.CreateCompilation("
+Namespace System
+    Public Class Object
+    End Class
+End Namespace
+Class Test
+End Class
+")
+
+            CompilationUtils.AssertTheseDiagnostics(comp, <expected>
+BC30002: Type 'System.Void' is not defined.
+    Public Class Object
+    ~~~~~~~~~~~~~~~~~~~~
+BC30183: Keyword is not valid as an identifier.
```

Comment:
Will remove it. No possibility of creating a user-defined Object in VB, or creating it in a C# reference without Void.

---
In reply to: [115349973](https://github.com/dotnet/roslyn/pull/18715#discussion_r115349973) [](ancestors = 115349973)